BEPs/BEP-336.md

```diff
@@ -333,16 +326,20 @@ For body retrieval responses (`BlockBodies`), the standard EIP-2718 blob transac
 Nodes MUST NOT automatically broadcast blob transactions to their peers.
 Instead, those transactions are only announced using `NewPooledTransactionHashes` messages, and can then be manually requested via `GetPooledTransactions`.
 
-#### BlockWithExtraData
+#### BlockPayload and P2P Message
 
-Unlike Ethereum, Blobs will be broadcasted with block as a unit for BSC, the broadcast message will be rlp encoded as:
+Unlike Ethereum, block & blobs are propagated atomically, either of the following approach is ok:
```

Comment:
>either of the following approach is ok

Is it saying both the approaches are included in bsc or only one of them?

Reply:
Right now, only approach 1 is supported, i.e. blob and blobs are in a single message.
But in the future, approach 2 can also be supported to improve the network efficiency.
I will refine the words